woocommerce/woocommerce-admin
# Added new notification to choose a niche

Fixes #4196

This PR adds a new notification to choose a niche.

### Detailed test instructions:
This notification should be visible for:
New users with `setup_client`: `false` on`woocommerce_onboarding_profile`
AND
one of the following values on `woocommerce_onboarding_profile`
`product_count`: `0`
`revenue`: `none`
`revenue`: `up to 2500`

To set these values it's necessary to:
- Go to the `Business Details` step (in the new onboarding).
![screenshot-one wordpress test-2020 05 25-15_51_04](https://user-images.githubusercontent.com/1314156/82837668-1c998300-9ea0-11ea-9fe2-6d26a3769af1.png)

- Set `How many products do you plan to display?` to `I don't have any products yet.`
- Set `What's your current annual revenue?` to `$0.00 (I'm just getting started)` or `Up to $2,500.00`.

- Verify the note is created and appears.
- Verify the action button works properly.



### Changelog Note:
Enhancement: Added new notification to choose a niche.
